h2o-core/src/main/java/hex/ModelMetrics.java

```diff
@@ -127,19 +127,25 @@ static public double getMetricFromModel(Key<Model> key, String criterion) {
   }
 
   static public double getMetricFromModelMetric(ModelMetrics mm, String criterion) {
-    if (null == criterion || criterion.equals("")) throw new H2OIllegalArgumentException("Need a valid criterion, but got '" + criterion + "'.");
+    if (null == criterion || criterion.equals(""))
+      throw new H2OIllegalArgumentException("Need a valid criterion, but got '" + criterion + "'.");
+
```

Comment:
this hunk seems to be reformat-only; wouldn't you consider adding curly braces?
Maybe my personal problem, but I find the python-like formatting slightly confusing in Java code.

Reply:
added